app/graph/types/webhook_type.rb

```diff
@@ -0,0 +1,23 @@
+class WebhookType < DefaultObject
+  description "Represents a Webhook, a BotUser with events and request url"
+
+  implements GraphQL::Types::Relay::Node
+
+  field :dbid, GraphQL::Types::Int, null: true
+  field :name, GraphQL::Types::String, null: true
+  field :events, GraphQL::Types::String, null: true
+  field :request_url, GraphQL::Types::String, null: true
+  field :headers, GraphQL::Types::String, null: true
+
+  def events
+    object.get_events.to_json
+  end
+
+  def request_url
+    object.get_request_url
+  end
+
+  def headers
+    object.get_headers.to_json
```

Comment:
As Sawy said above for the `BotUser` model, if the fields are defined as `JsonStringType`, you don't need to call `.to_json` here.